applications/wg-easy/charts/wg-easy/values.yaml

```diff
@@ -1,40 +1,152 @@
+          WG_PERSISTENT_KEEPALIVE: '{{ dig "wireguard" "persistentKeepalive" "" .Values }}'
+          WG_DEFAULT_ADDRESS: '{{ dig "wireguard" "defaultAddress" "" .Values }}'
+          WG_DEFAULT_DNS: '{{ dig "wireguard" "defaultDns" "" .Values }}'
+          WG_ALLOWED_IPS: '{{ dig "wireguard" "allowedIps" "" .Values }}'
+          WG_PRE_UP: '{{ dig "wireguard" "preUp" "" .Values }}'
+          WG_POST_UP: '{{ dig "wireguard" "postUp" "" .Values }}'
+          WG_PRE_DOWN: '{{ dig "wireguard" "preDown" "" .Values }}'
+          WG_POST_DOWN: '{{ dig "wireguard" "postDown" "" .Values }}'
+        envFrom:
+          - secretRef:
+              identifier: webpass
+        image:
+          repository: ghcr.io/wg-easy/wg-easy
+          tag: 14.0
+          pullPolicy: IfNotPresent
+        ports:
+          - containerPort: 51821
+            protocol: TCP
+          - containerPort: 51820
+            protocol: UDP
+        resources:
+          requests:
+            cpu: 50m
+            memory: 50Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            add:
+              - NET_ADMIN
+              - SYS_MODULE
+        probes:
+          liveness:
+            enabled: true
+            spec:
+              initialDelaySeconds: 0
+              periodSeconds: 10
+              timeoutSeconds: 5
+              failureThreshold: 5
+          readiness:
+            enabled: true
+            spec:
+              initialDelaySeconds: 0
+              periodSeconds: 10
+              timeoutSeconds: 1
+              failureThreshold: 5
+secrets:
+  webpass:
+    enabled: true
+    stringData:
+      PASSWORD_HASH: '{{ dig "wireguard" "password" "$2a$12$Zh4e0Lnbfhh3.EJTLKPApOCOZgjmaWm0eOMDI7YkoACKMjKPFs7.a" .Values }}'
+service:
+  web:
+    controller: wg-easy
+    enabled: true
+    type: ClusterIP
+    ipFamilyPolicy: SingleStack
+    ipFamilies:
+      - IPv4
+    ports:
+      http:
+        enabled: true
+        port: 51821
+        protocol: HTTP
+    primary: true
+  vpn:
+    controller: wg-easy
+    enabled: true
+    type: NodePort
+    ipFamilyPolicy: SingleStack
+    ipFamilies:
+      - IPv4
+    ports:
+      udp:
+        enabled: true
+        port: 51820
+        protocol: UDP
+persistence:
+  config:
+    enabled: true
+    type: persistentVolumeClaim
+    storageClass: ''
+    accessMode: ReadWriteOnce
+    size: 1Gi
+    retain: false
+    globalMounts:
+      - path: /etc/wireguard
+
+# Troubleshoot
+troubleshoot:
+  support-bundles:
+    replicated:
+      enabled: true
+    # wg-easy:  # arbitrary name for your custom spec
+    #   enabled: true
+    #   collectors:
+    #     - logs:
+    #         name: wg-easy
+    #         collectorName: wg-easy
+    #         selector:
+    #           - app=wg-easy
+    #         namespace: '{{ .Release.Namespace }}'
+    #         containerNames:
+    #           - wg-easy
```

Comment:
Does the SDK process this or something? I don't see what this is configuring.

Reply:
This was actually a holdover from the replicated-library chart where we generated support bundles, I believe; so I think we can take this out